packages/web-runtime/src/helpers/theme.js

```diff
@@ -0,0 +1,22 @@
+import defaultTheme from 'web-runtime/themes/owncloud.json'
+export const loadTheme = async (name = '') => {
+  const defaults = { theme: defaultTheme, name: 'themes/owncloud/theme.json' }
```

Comment:
`themes/owncloud/theme.json` doesn't resolve

Reply:
(irrelevant, since we don't use `loadTheme` with an empty name)

Reply:
Ah, thanks for explaining that this name doesn't need to resolve. Only for having a proper key.

Reply:
it's just the name, it does not need to resolve